src/pkg/packaging/deb/package.targets

```diff
@@ -41,8 +46,13 @@
                             BuildDebs;"
           Condition="'$(BuildDebPackage)'=='true'" />
 
+  <!-- Build only on Ubuntu14.04 -->
+  <Target Name="GenerateRuntimeDebs" 
+          DependsOnTargets="$(RuntimeDebsDependsOn)"
+          Condition="'$(PackageTargetRid)' == 'ubuntu.14.04-x64'" />
```

Comment:
Why encode this into the build scripts?  That makes it hard for someone who only has an ubuntu 16 machine develop and test in this area.

Instead, I think we should have a boolean property here that gets passed in by the build definition.